src/main/java/net/pms/network/RequestHandler.java

```diff
@@ -94,8 +94,17 @@ public void run() {
 			// header matches are attempted and if those fail as well we're stuck with the
 			// default renderer.
 
-			// Attempt 1: try to recognize the renderer by its socket address from previous requests
-			renderer = RendererConfiguration.getRendererConfigurationBySocketAddress(ia);
+			// Attempt 1: If the reguested url contains the no-transcode tag, force
+			// the default streaming-only conf.
+			if (request.getArgument().contains(RendererConfiguration.NOTRANSCODE)) {
+				renderer = RendererConfiguration.getStreamingConf();
+				LOGGER.info("Forcing streaming.");
```

Comment:
@skeptical I think it should be set to```debug``` or better to ```trace``` not to the ```info```. It spams the info whenever the chunk is asked to stream. The same should be applied for ```RequestHandlerV2```